src/main/java/com/codeborne/selenide/collections/Attributes.java

```diff
@@ -0,0 +1,41 @@
+package com.codeborne.selenide.collections;
+
+import com.codeborne.selenide.impl.Html;
+import org.openqa.selenium.WebElement;
+
+import javax.annotation.CheckReturnValue;
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.List;
+
+@ParametersAreNonnullByDefault
+public class Attributes extends ExactAttributes {
+  public Attributes(String attribute, String... expectedValues) {
+    super(attribute, expectedValues);
+  }
+
+  public Attributes(String attribute, List<String> expectedValues) {
+    super(attribute, expectedValues);
+  }
+
+  @CheckReturnValue
+  @Override
+  public boolean test(List<WebElement> elements) {
+    if (elements.size() != expectedValues.size()) {
+      return false;
+    }
+
+    for (int i = 0; i < expectedValues.size(); i++) {
+      WebElement element = elements.get(i);
+      String expectedValue = expectedValues.get(i);
+      if (!element.getAttribute(attribute).contains(expectedValue)) {
```

Comment:
`element.getAttribute()` is nullable and will cause NPE here

Reply:
Fixed
```java
  protected String getAttributeValue(WebElement element) {
    String attr = element.getAttribute(attribute);
    return attr == null ? "" : attr;
  }

```

Added Tests for not existing attribute Behaviour